paddle/gserver/gradientmachines/NeuralNetwork.cpp

```diff
@@ -309,35 +309,35 @@ class CombinedEvaluator : public Evaluator {
   void addEvaluator(std::unique_ptr<Evaluator>&& evaluator) {
     evaluators_.emplace_back(std::move(evaluator));
   }
-  virtual void start() {
+  void start() override {
```

Comment:
I don't get it -- why is keyword `override` a key to Mac OS X build? @reyoung @jacquesqiao